lib/shopify-cli/commands/config.rb

```diff
@@ -7,6 +7,7 @@ class Config < ShopifyCli::Command
 
       subcommand :Feature, "feature"
       subcommand :Analytics, "analytics"
+      subcommand :ArgoAdminBeta, "argo-admin-beta"
```

Comment:
@tanema can probably correct me if I'm wrong on this one, but AFAIK the point behind that feature is to allow us to easily add toggles to the config file.

Reply:
cc @andyw8 too if you have any other suggestion.

Reply:
@henrytao-me and I paired on this, and it looks like we can just use `config feature argo-admin-beta --enable`, rather than adding a new type of feature here.

Reply:
@paulomarg shows me how to use `feature`. I will switch to use it. `shopify config feature argo-admin-beta --enable`